### DISCUSSION TITLE
[A/L] (2021/22) Induzione - Esercizio 12

### DISCUSSION THREAD

**Discussion by matypist** [sapienzastudentsnetwork/metodi-matematici-per-l-informatica#312] [Esercizi A-L]
![image](https://user-images.githubusercontent.com/69389461/152322497-620d3b53-0b6f-4c43-8b43-feb87eb28d77.png)

**Comment by Exyss** [comment]
![Esercizio 12)](https://user-images.githubusercontent.com/65043207/152659543-d9c55009-7230-44fd-bb1d-f2ec5085c77c.jpg)
